src/main/resources/templates/signup.html

```diff
@@ -0,0 +1,71 @@
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
+    <title>Moody Bluez</title>
+    <link href="/css/signup.css" rel="stylesheet">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css"/>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;500;600;700&display=swap" rel="stylesheet">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+    <link href="/css/style.css" rel="stylesheet">
+
+</head>
+<body>
+
+<div th:replace="fragments/topnav :: navbar">
+
+</div>
+
+<form action="#" th:action="@{/process_register}" th:object="${user}" enctype="multipart/form-data"  method="POST"><br />
+    <div class="row">
+        <div class="col">
+
+        </div>
+        <div class="col">
+
+
+    <div class="input-group mb-4">
+        <div class="input-group-prepend">
+            <span class="input-group-text" id="Username">Username</span>
+        </div>
+        <input type="text" class="form-control" aria-label="Sizing example input" aria-describedby="inputGroup-sizing-sm" name="Username" th:field="*{userName}"/>
+    </div>
+    </div>
+        <div class="col">
+
+        </div>
+    </div>
+    <div class="row">
+        <div class="col">
+        </div>
+        <div class="col">
+
+
+    <div class="input-group mb-4">
+        <div class="input-group-prepend">
+            <span class="input-group-text" id="Password">Password</span>
+        </div>
+        <input type="text" class="form-control" aria-label="Sizing example input" aria-describedby="inputGroup-sizing-sm" name="Password" th:field="*{password}"/>
+    </div>
+    </div>
+        <div class="col">
+        </div>
+    </div>
+
+    <div class="row">
+    <div class="col"></div>
+        <div class="col">
+        <button class="btn btn-primary" name="submit" type="submit" value="0">Submit</button>
+        </div>
+        <div class="col"></div>
+    </div>
+</form>
+
+</body>
+
```

Comment:
I added the signup form to its own page and used bootstrap to format and align the components. I moved this from the original page because it was not needed there.